SU2_CFD/src/numerics/NEMO/convection/msw.cpp

```diff
@@ -198,7 +198,7 @@ CNumerics::ResidualType<> CUpwMSW_NEMO::ComputeResidual(const CConfig *config) {
                                                       epsilon*epsilon));
 
   /*--- Compute projected P, invP, and Lambda ---*/
-  CreateBasis(UnitNormal);
+  CreateBasis   (UnitNormal);
   GetPMatrix    (Ust_i, Vst_i, dPdUst_i, UnitNormal, l, m, P_Tensor   );
```

Comment:
You will not be pleased when we eventually run clang-format... :/

Reply:
Yeah....hahah. But ill change this back.